Dockerfile

```diff
@@ -0,0 +1,61 @@
+# Install dependencies only when needed
+FROM node:16-alpine AS deps
```

Comment:
Let's not use Alpine as our base. It uses **glibc**. Instead we can use the standard NODE instance

Reply:
as per the doc of node-alpine it uses musl libc instead of glibc and friends. and in nextjs official doc example also it uses alpine: https://nextjs.org/docs/pages/building-your-application/deploying#:~:text=Clone%20the%20with%2Ddocker%20example